examples/11_advanced_parametric_study.py

```diff
@@ -116,8 +116,8 @@
 # ---------------------
 # If you a working with a large parametric study, you may want to skip some
 # simulations to reduce processing time. To do so, set the simulation status
-# to ``SimulationStatus.SKIP`` which is defined in the ``SimulationStatus``
-# class. Here, a ``Pandas DataFrame`` is obtained, a filter is
+# to :obj:`SimulationStatus.SKIP` which is defined in the :class:`SimulationStatus`
+# class. Here, a :class:`~pandas.DataFrame` is obtained, a filter is
```

Comment:
```suggestion
# to :obj:`SimulationStatus.SKIP` which is defined in the :class:`SimulationStatus`
# class. Here, a :class:`~pandas.DataFrame` object is obtained, a filter is
```
Here, the "which is defined in the :class:`SimulationStatus` class modifies simulation status. Thus, the sentence should be changed as follows:

To do so, set the simulation status, which is defined in the :class:`SimulationStatus` class, to :obj:`SimulationStatus.SKIP`. 

Of course, after the code entity, there should be a noun, but I don't know how to do that in this case!